### ISSUE TITLE
Cannot use anonymous data matrix as verification data

### ISSUE THREAD

**Issue by diogofernandes63** [jpmml/sklearn2pmml#161]
Hi,

I am running into an error when running sklearn2pmml using the following code. The preprocessing code transforms the data fine but it's giving me an error.

Below it's my code.
[code1.txt](https://github.com/jpmml/sklearn2pmml/files/3296950/code1.txt)

And when I try to (sklearn2pmml(pipeline,"model2.pmml",with_repr= True,debug=True))
I get
[error.txt](https://github.com/jpmml/sklearn2pmml/files/3296952/error.txt)

**Comment by vruusmann** [comment]
Your pipeline does not contain feature and/or label column names. Hence, there's a problem establishing a relationship between training data columns and verification data columns, which leads to this NPE being thrown.

Recommended solution - add column names to your datasets. Alternative solution - comment out the `PMMLPipeline.verify(X)` line

**Comment by vruusmann** [comment]
Was fixed in https://github.com/jpmml/jpmml-sklearn/commit/0400175a2ffc05384fb47bcc67431b8da4d0093b
